test/obb/events/game_events_test.clj

```diff
@@ -21,29 +21,20 @@
   (let [system (-> (game-events-system) component/start)
```

Comment:
Well, honestly for me everything is more readable with `->`. :) Anyway is not idiomatic clojure do use is with just two actions. I'll change it.